src/hdmf_zarr/backend.py

```diff
@@ -167,6 +167,15 @@ def file(self):
         """
         return self.__file
 
+    @property
+    def _file(self):
```

Comment:
Instead of adding this property, should `def file` be renamed instead? At the very least, I would recommend that this property should `return self.file` instead to make sure that any logic that may be added later doesn't get duplicated or that the two functions don't get out-of-sync.

Reply:
> all operations should go through `HDF5IO` to keep the two in sync.

Doe you mean `HDMFIO` here?

> `HDF5IO` does not publicly expose the `h5py.File` object

I think `HDF5IO` has the same `_file` property 

> For the same reason, I would say `ZarrIO` should not expose the private open group.

I think exposing them as a "private" `_file` property that is intended for internal use is fine. At the very least this is useful for testing. However, I agree that it should not be a public property.